flink/v1.12/flink/src/test/java/com/netease/arctic/flink/FlinkTestBase.java

```diff
@@ -105,66 +102,38 @@ public class FlinkTestBase extends TableTestBase {
   protected Map<String, String> props;
   private volatile StreamExecutionEnvironment env = null;
 
+  public static final Schema TABLE_SCHEMA = new Schema(
+    Types.NestedField.required(1, "id", Types.IntegerType.get()),
+    Types.NestedField.required(2, "name", Types.StringType.get()),
+    Types.NestedField.required(3, "ts", Types.LongType.get()),
+    Types.NestedField.required(4, "op_time", Types.TimestampType.withoutZone())
+  );
+
   public static final TableSchema FLINK_SCHEMA = TableSchema.builder()
-      .field("id", DataTypes.INT())
-      .field("name", DataTypes.STRING())
-      .field("op_time", DataTypes.TIMESTAMP())
-      .build();
+    .field("id", DataTypes.INT())
```

Comment:
You can also have a subclass extending `BasicTableTestHelper` which can have its own TABLE_SCHEMA so you can keep the same schema as before.

Reply:
thanks, now use BasicTableTestHelper.TABLE_SCHEMA can be aligned with other modules.